JSDemos/Demos/Scheduler/Agenda/description.md

```diff
@@ -1 +1,7 @@
-This demo illustrates the **agenda** view, which lists all available appointments grouped by&nbsp;a&nbsp;date. Designed for narrow screens, this view represents appointments compactly.
\ No newline at end of file
+The Agenda view shows appointments as a simple list. This is the most compact display style because it doesn't render a time scale. Appointments follow each other without empty spaces and their size doesn't depend on event duration.
```

Comment:
`Appointments follow each other without empty spaces and their size doesn't depend on event duration.`

Does each appointment have it's own size? If so, should it say "and their sizes do not depend on event duration".